packages/devtools_app/test/vm_developer/vm_developer_test_utils.dart

```diff
@@ -96,6 +96,14 @@ final testRetainingObjects = [
     parentListIndex: 1,
     parentField: 'fooParentField',
```

Comment:
This test is used to check which property takes priority between `parentListIndex` and `parentField`, but it should be impossible for both of these properties ot appear on the same `RetainingObject`. Do we want to change this?

Reply:
Yeah, if it doesn't make sense anymore, we should update the test.